picamera2/previews/drm_preview.py

```diff
@@ -150,23 +150,24 @@ def render_drm(self, picam2, completed_request):
 
             self.plane = self.resman.reserve_overlay_plane(self.crtc, format=fmt)
```

Comment:
Are we happy that this will return None and won't raise an exception in all cases?

Reply:
Not checked, but needs fixing if it does.
I'm certainly not seeing an exception raised if there is no overlay plane.

Reply:
Perhaps I'll just ask our OP to give this a try, and go with it if it appears to work!